.gitlab-ci.yml

```diff
@@ -91,7 +91,12 @@ publish:s3:latest:
   extends: publish:s3
   only:
     variables:
-      - $MENDER_VERSION =~ /[0-9]+\.[0-9]+\.[0-9]+/
-  before_script:
-    - apt update && apt install -yyq awscli
-    - deb_version=latest
+      - $PUBLISH_LATEST_CLIENT_PACKAGE == "true"
```

Comment:
Please add the variable (optionally with a doc string) at the beginning of the pipeline, in `variables` section, with default falue `false`. Keep in mind that we might need to trigger it manually some times so it is good to have it at the top.